src/main/kotlin/com/sourcegraph/cody/ui/CodyToolWindowFactory.kt

```diff
@@ -298,10 +299,16 @@ class WebUIHostImpl(
     val id = decodedJson?.get("id")?.asString
     val arg = decodedJson?.get("arg")?.asString
 
-    if (command == "auth") {
-      val authKind = decodedJson.get("authKind")?.asString
-      if (authKind == "signout") {
-        CodyAuthenticationManager.getInstance(project).setActiveAccount(null)
+    if ((command == "auth" && decodedJson.get("authKind")?.asString == "signout" ||
```

Comment:
why do we have the extra set of parens around this expression?

Reply:
It was actually a (harmless) bug, fixed.